docs/generate_docs.py

```diff
@@ -155,7 +153,7 @@ def clone_from_github(repo, tag, org):
         subprocess.run(clone_command, check=True)
         log_message(f"Successfully cloned {repo}")
     except subprocess.CalledProcessError as e:
-        log_message(f"Failed to clone {repo}. Error: {e}")
+        raise (e)
```

Comment:
Does `e` already include the repo's info?

Reply:
Yes, the exception in the job I linked looks like this: 
```bash
subprocess.CalledProcessError: Command '['git', 'clone', '--branch', '', '--single-branch', 'https://github.com/triton-inference-server/perf_analyzer.git']' returned non-zero exit status 128.
```
This should be sufficient to understand why the script failed.